static/app/views/settings/featureFlags/newSecretHandler.tsx

```diff
@@ -0,0 +1,68 @@
+import type {MouseEventHandler} from 'react';
+import styled from '@emotion/styled';
+
+import Alert from 'sentry/components/alert';
+import {Button} from 'sentry/components/button';
+import FieldGroup from 'sentry/components/forms/fieldGroup';
+import PanelItem from 'sentry/components/panels/panelItem';
+import TextCopyInput from 'sentry/components/textCopyInput';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+function NewSecretHandler({
+  secret,
+  handleGoBack,
+}: {
+  handleGoBack: MouseEventHandler;
+  secret: string;
+}) {
+  return (
+    <div>
+      <Alert type="warning" showIcon system>
+        {t('The secret has been posted.')}
+      </Alert>
+
+      <PanelItem>
+        <InputWrapper>
+          <FieldGroupNoPadding
+            label={t('Secret')}
+            help={t(
+              'The secret should not be shared and will not be retrievable once you leave this page.'
+            )}
+            inline
+            flexibleControlStateSize
+          >
+            <TextCopyInput aria-label={t('Secret')}>{secret}</TextCopyInput>
+          </FieldGroupNoPadding>
+        </InputWrapper>
+      </PanelItem>
+
+      <PanelItem>
+        <ButtonWrapper>
+          <Button onClick={handleGoBack} priority="primary">
+            {t('Done')}
+          </Button>
+        </ButtonWrapper>
+      </PanelItem>
+    </div>
+  );
+}
+
+const InputWrapper = styled('div')`
+  flex: 1;
+`;
+
+const FieldGroupNoPadding = styled(FieldGroup)`
+  padding: 0;
+`;
+
+const ButtonWrapper = styled('div')`
+  margin-left: auto;
```

Comment:
Does this do anything?

Reply:
this is what it looks like without it:

<img width="992" alt="SCR-20241204-jmew" src="https://github.com/user-attachments/assets/bfdd0939-7027-4aec-b2f0-0afed4a415b5">

`margin-left: auto` makes the button go to the right